e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/validation/company-details-alternative-trading-name-validation.spec.js

```diff
@@ -0,0 +1,58 @@
+import { submitButton, yesRadioInput, field } from '../../../../../../../pages/shared';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { ROUTES, FIELD_IDS } from '../../../../../../../constants';
+
+const {
+  EXPORTER_BUSINESS: {
+    YOUR_COMPANY: {
+      ALTERNATIVE_TRADING_NAME,
+    },
+  },
+} = FIELD_IDS.INSURANCE;
+
+const COMPANY_DETAILS_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+
+describe("Insurance - Your business - Company details page- As an Exporter I want to enter details about my business in 'your business' section - alternative trading name validation", () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+
+      url = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.EXPORTER_BUSINESS.COMPANY_DETAILS}`;
```

Comment:
Can we destructure the routes above, so we can consume `COMPANY_DETAILS` here, and have a `baseUrl` const